packages/app/components/profile/profile.web.tsx

```diff
@@ -298,14 +298,10 @@ const Profile = ({ username }: ProfileScreenProps) => {
                 useWindowScroll={isMdWidth}
                 ListHeaderComponent={Header}
                 numColumns={1}
+                preserveScrollPosition
```

Comment:
didn't intend to add an extra prop. But it was getting tricky with many lists in single page. I was assuming that we'd require scroll position saving just for window scroll lists, but we're not using window scroll on mobile web, so we need to save scroll position for window and normal virtualiser  😕

Reply:
Why can't we actually use window scroll on mobile? With the new dvh it should be pretty doable, or?

Reply:
i think yes, it should be doable. I am not sure what was the exact reason of removing it. @alantoa would know better. if we do that, we can remove this prop for sure and just support scroll position for window lists.

Reply:
it can be done in a separate PR though!